pygeoapi/provider/oracle.py

```diff
@@ -804,6 +817,7 @@ def query(
                     q,
                     language,
                     filterq,
+                    extra_params=extra_params
```

Comment:
I think we should add something about these extra parameters for Oracle in the [documentation of the provider](https://github.com/geopython/pygeoapi/blob/master/docs/source/data-publishing/ogcapi-features.rst#oracle).